minitests/roi_harness/Makefile

```diff
@@ -1,16 +1,25 @@
 BUILD_DIR=build
+HARNESS_DIR ?= harness
 export BUILD_DIR
 
 XRAY_PINCFG ?= BASYS3-SWBUT
 export XRAY_PINCFG
 
+HARNESS_FILES=$(BUILD_DIR)/design.bit $(BUILD_DIR)/design.txt $(BUILD_DIR)/design.json $(BUILD_DIR)/design.dcp
+
 # official demo configuration
-all:
+all: $(HARNESS_FILES)
+
+$(HARNESS_FILES): runme.sh runme.tcl
 	./runme.sh
 
+copy: $(HARNESS_FILES)
+	mkdir -p $(HARNESS_DIR)
+	cp $(HARNESS_FILES) $(HARNESS_DIR)
+
 clean:
```

Comment:
clean should now be
clean:
rm -rf $(BUILD_DIR) $(HARNESS_DIR)

Don't think harness dir is being removed, and the other stuff doesn't need to be there anymore?